apps/portals/my-pages/src/components/Greeting/Greeting.tsx

```
@@ -8,13 +7,14 @@
   Text,
 } from '@island.is/island-ui/core'
 import { useLocale } from '@island.is/localization'
-import { useAuth } from '@island.is/auth/react'
+import { useUserInfo } from '@island.is/react-spa/bff'
 import { m } from '@island.is/service-portal/core'
+import React, { FC } from 'react'
 import * as styles from './Greeting.css'
 
 const Greeting: FC<React.PropsWithChildren<{}>> = () => {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)